lib/services/hpcpackage_remain.pm

```diff
@@ -89,10 +90,14 @@ sub full_pkgcompare_check {
     my $stage = $hash{stage};
 
     if ($stage eq 'before') {
+        # It need python2 module to compare packages
+        add_suseconnect_product("sle-module-python2", undef, undef, undef, 300, 1) if (check_var('DROPPED_MODULES', 'python2'));
```

Comment:
I think the condition need to update since later we may have another module dropped, if it is 'python2, xxx' then can't fulfill the condition.

Reply:
This only need python2. Just for python2 workaround. I'm not sure later if it will be impacted or not. But I think it doesn't need consider so much at here. Because it is just a workaround for special case.